### PR TITLE
Upcoming vote

### PR DESCRIPTION
**Pull request by lubej**

### CODE REVIEW

**Review by buberdds — APPROVED** [review]

File: frontend/src/pages/UpcomingVotePage/index.module.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    font-weight: 400;
+    line-height: 130%;
+    letter-spacing: -0.03em;
+  }
+}
+
+@media screen and (min-width: 1000px) {
+  .mascotCard {
+    &.mascotCard {
+      &.mascotCard {
```
[the remark this answers is not in the document]
Comment (lubej) [reply frontend/src/pages/UpcomingVotePage/index.module.css:66]: It overrides the MascotCard css, without using `!important`. It's fine.

[the remark this answers is not in the document]
Comment (buberdds) [reply frontend/src/pages/UpcomingVotePage/index.module.css:66]: I am Ok with merging but imo it's not fine from a design/maintenance aspect. MascotCard should have a prop that is used in UpcomingVoting and all styling of mascotCard should be placed in proper css module.

[the remark this answers is not in the document]
Comment (lubej) [reply frontend/src/pages/UpcomingVotePage/index.module.css:66]: This page should go away, anyway. It was not planned ahead.

But, I would make modification to the MascotCard, just for single css prop. You can pass className, which is enough IMO.

[the remark this answers is not in the document]
Comment (lubej) [reply frontend/src/pages/UpcomingVotePage/index.module.css:66]: As for separation of concern, not sure. Maybe it makes more sense, to be in MascotCard. But from code perspective, it is a single usage, only in `UpcomingVotePage` component.

**Pull request merged**